MdsSupportingClasses/Collivery.php

```diff
@@ -1306,4 +1300,29 @@ public function getCurrentUsernameAndPassword()
             'password' => $this->config->user_password,
         ];
     }
+
+    /**
+     * @param Array $data - Contains the array you want to modify
+     * @param string $key - This is the name of the Id field
+     * @param string $value - This is the name of the Value field
+     * 
+     * @return Array $key_value_array - {key:value, key:value} - Used for setting up dropdown lists.
+     */
+    function make_key_value_array($data, $key, $value, $isContact = false) {
```

Comment:
Please define the scope of this method (`private`/`public`)

Reply:
updated scope to public.